### ISSUE TITLE
DONT WRITE BACK TO YOUR BIBTEX-FILE: custom fields are imported with column-names that includes the values in the  fields...!!

### ISSUE THREAD

**Issue by emilBeBri** [ropensci/bibtex#38]
So I've been trying to use this function for a while, but when using it and writing the file back to disk with df2bib(), Jabref suddenly told me

![scrot_2021-03-03_20-54-10](https://user-images.githubusercontent.com/964816/109867839-453c7d80-7c67-11eb-984e-16bd6d1e66d7.png)

I found this puzzling, and tried troubleshooting the error.  While doing this, I then stumbled upon something where using the eprint field, a entry that looked like this (very minimal)

```
@inproceedings{joulin2017bag,
    title = "Bag of Tricks for Efficient Text Classification",
    author = "Joulin, Armand",
    year = "2017",
    eprint={1607.01759},
}
```
was transformed into this

```
@Inproceedings{joulin2017bag,
  Author = {Joulin, Armand},
  Title = {Bag of Tricks for Efficient Text Classification},
  Year = {2017},
  Eprint..1607.01759.. = {1607.01759}
}
```
This problem does not appear to related to issue with the empty text fields that I was actually investigating. So there are two serious bugs here, and only one of them is immediatly reproducible.   

I conclude that this package should be used with caution, and you should definitely NOT write back into a bibtex file that you need for anything reliable.

**Comment by GeoBosh** [comment]
Please note that your input `bibtex` entry is invalid - for entries of type `inproceedings` the field `booktitle` is compulsory for the standard `bibtex` styles. Also, I am surprised that `bibtex` even parsed it, since my slightly older version of package bibtex, v0.4.2.2, gives this:
```
> bibtex::read.bib("bibtex_issue.bib")
ignoring entry 'joulin2017bag' (line 1) because :
	A bibentry of bibtype ‘InProceedings’ has to specify the field: booktitle
```
Please note that the tone of your posting is insulting, maybe unintentionally. You have given a good description of your problem, perceived or not, for the maintainers (I am not among them) to look at. And that should have been it.

**Comment by emilBeBri** [comment]
I don't agree with you that it's insultating - to you, perhaps, but not generally.  Of course that's not my intention: I'm stating that this bug seems to be so severe, that it should not be used to write a .bib-file, since this bug will in all likehood scramble the original  bib-input. This seems to me to be aa very important point to get through, before anybody wrecks  something important.

**Comment by mwmclean** [comment]
The ALL CAPS and double exclamation points in the title... so authoritative and you couldn't even bother to create a reproducible bug report. Your post is 'insultating' to any developer. I conclude your GitHub comments "shouldn't be used" EXCLAMATION MARKS!!!!!!

**Comment by emilBeBri** [comment]
This bug nearly ruined an important .bib file i was using, so the all caps and !! was with the intention of making sure nobody else ruined something important they were doing. 

I find your suggestion that I "didn't bother including a reprex" to be rude - I spend quite a bit of timing figuring out what the error was, so I could post it here.

**Comment by emilBeBri** [comment]
But of course, if two different people both thinks I wrote the report in a rude manner, I will take that into consideration. There was no bad faith involved, however. The all caps and exclamation mark was probably a bad idea.

**Comment by dieghernan** [comment]
Leaving apart other discussions, I think there  is a fair point on here. I would propose to implement a mechanism for creating a backup of the bib file just in case something goes wrong. I already did this on **cffr**:

https://github.com/ropensci/cffr/blob/855608c8f6d3d7da0180db3e978143a288b6558b/R/write_bib.R#L65:L79
